src/test/scala/is/hail/methods/LDPruneSuite.scala

```diff
@@ -150,8 +152,33 @@ class LDPruneSuite extends SparkSuite {
         val gs2 = convertGtsToGs(v2)
         val bv1 = LDPrune.toBitPackedVector(gs1.hardCallIterator, nSamples)
         val bv2 = LDPrune.toBitPackedVector(gs2.hardCallIterator, nSamples)
-        val sgs1 = RegressionUtils.normalizedHardCalls(gs1, nSamples).map(math.sqrt(1d / nSamples) * BVector(_))
-        val sgs2 = RegressionUtils.normalizedHardCalls(gs2, nSamples).map(math.sqrt(1d / nSamples) * BVector(_))
+
+        val rowType = TStruct(
+          "l" -> TLocus(GenomeReference.GRCh37),
+          "v" -> TVariant(GenomeReference.GRCh37),
+          "va" -> TStruct(),
+          "gs" -> TArray(TGenotype)
+        )
+        def makeRV(gs: Iterable[Genotype]): RegionValue = {
+          val gArr = gs.toArray
+          val rvb = new RegionValueBuilder(MemoryBuffer())
+          rvb.start(rowType)
+          rvb.startStruct()
+          rvb.setMissing()
+          rvb.setMissing()
+          rvb.setMissing()
+          rvb.startArray(gArr.length)
+          gArr.foreach { g => rvb.addAnnotation(TGenotype, g) }
```

Comment:
You can write this as  `rvb.addAnnotation(gArr)`, but you'll need to change the return type for `convertGtsToGs` to `IndexedSeq[Genotype]`.